examples/upbound-aws-provider/composite-resources/serverless-examples/microservice/README.md

```diff
@@ -0,0 +1,337 @@
+xapigateways.awsblueprints.io       True          True      24m
+xdynamodbtables.awsblueprints.io    True          True      24m
+xlambdafunctions.awsblueprints.io   True          True      24m
+xmicroservices.awsblueprints.io     True          True      24m
+xnotifications.awsblueprints.io     True          True      24m
+```
+
+Verify the Compositions
+```
+kubectl get compositions
+```
+Expected output
+```
+NAME                                            XR-KIND           XR-APIVERSION               AGE
+container.lambda.aws.upbound.awsblueprints.io         XLambdaFunction   awsblueprints.io/v1alpha1   23m
+lambda-authorizer-invoke.iampolicy.awsblueprints.io   IAMPolicy         awsblueprints.io/v1alpha1   23m
+lambda-invoke-apigw.iampolicy.awsblueprints.io        IAMPolicy         awsblueprints.io/v1alpha1   23m
+lambda-invoke.iampolicy.awsblueprints.io              IAMPolicy         awsblueprints.io/v1alpha1   23m
+microservices.upbound.awsblueprints.io                XMicroservice     awsblueprints.io/v1alpha1   23m
+read-kms.iampolicy.awsblueprints.io                   IAMPolicy         awsblueprints.io/v1alpha1   23m
+read-s3.iampolicy.awsblueprints.io                    IAMPolicy         awsblueprints.io/v1alpha1   23m
+read-sqs.iampolicy.awsblueprints.io                   IAMPolicy         awsblueprints.io/v1alpha1   23m
+rest.apigateway.aws.upbound.awsblueprints.io          XApiGateway       awsblueprints.io/v1alpha1   23m
+s3.lambda.aws.upbound.awsblueprints.io                XLambdaFunction   awsblueprints.io/v1alpha1   23m
+sns.notification.upbound.awsblueprints.io             XNotification     awsblueprints.io/v1alpha1   23m
+table.dynamodb.awsblueprints.io                       XDynamoDBTable    awsblueprints.io/v1alpha1   23m
+write-cloudwatch-metrics.iampolicy.awsblueprints.io   IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-dynamodb.iampolicy.awsblueprints.io             IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-firehose-s3.iampolicy.awsblueprints.io          IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-firehose.iampolicy.awsblueprints.io             IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-s3.iampolicy.awsblueprints.io                   IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-sqs.iampolicy.awsblueprints.io                  IAMPolicy         awsblueprints.io/v1alpha1   23m
+```
+
+### Build Lambda function packages
+
+This example uses ZIP packaging of the Lambda functions. 
+ - Navigate to `./src/logic/` directory
+ - Create ZIP file `microservice-business-logic.zip` with the `lambda_function.py` in it:
+```shell
+zip microservice-business-logic.zip lambda_function.py
+```
+ - Navigate to `./src/authorizer/` directory
+ - Create ZIP file `microservice-authorizer.zip` with the `lambda_function.py` in it:
+```shell
+zip microservice-authorizer.zip lambda_function.py
+```
+ - Create Amazon S3 bucket
+ - Upload both ZIP files to the S3 bucket you just created
```

Comment:
You should explain the S3 key requirement. Better to provide an example of `aws s3` command.

Reply:
Updated